### ISSUE TITLE
[High] Chrome+JAWS: Keyboard: 'Mark as unread' and 'Delete comment' controls are not accessible using the keyboard.

### ISSUE THREAD

**Issue by SumitDiyora** [Expensify/App#8413]
## Action Performed:
1. Using Chrome+JAWS, open URL: staging.new.expensify.com
2. Navigate to any of the chats with the attached image and hard press on the image.
3. Observe 'Mark as unread' and 'Delete comment' controls are accessible using the keyboard or screen reader users.

## Expected Result:
Keyboard should be accessible for 'Mark as unread' and 'Delete comment' controls.

## Actual Result:
'Mark as unread' and 'Delete comment' controls can be activated using a mouse but it's not activated using the keyboard.

**Other Occurrences:**
>Similar type of issue repro on PRs #8195, #8117, #8340, #8443, #8585, #8672, #8815, #9025, #9026, #8569 for 'Copy to clipboard' and 'Edit comment' controls.
>Similar type of issue repro on PR #8980 -Request a call is not accessible using the keyboard.

## Workaround:
Yes

## Area issue was found in:
Any of the chats with the attached image

## Failed WCAG checkpoints
2.1.1

## User impact:
Keyboard only users will not be able to navigate to the controls via TAB or SHIFT+TAB keys

## Suggested resolution:
Ensure that all controls provide keyboard focus and are operable without the use of a mouse. It is recommended to use native HTML controls such as <button> instead of custom controls as they provide keyboard focus and visible outline focus by default. If custom controls must be used, tabindex="0" can be provided for focus.

## Platform:
- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** 1.1.49-0
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**Reference link:** https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button
**Issue reported by:** Sumit

https://user-images.githubusercontent.com/98817464/161129309-0ecb7a74-bef4-46fe-b392-22c5988267fb.mp4
